single_tech_samples/datafactory/sample2_data_pre_processing_with_azure_batch/deploy/adf/deploy-pipeline.sh

```diff
@@ -0,0 +1,13 @@
+AZ_RESOURCE_GROUP="<YOUR-RESOURCEGROUP-NAME>"
```

Comment:
add "#!/bin/sh"

Reply:
When cloning this repo on MAC, this script doesn't have execute privilege, please add that.
```
chmod +x deploy-pipeline.sh
```